confgenerator/built-in-config-windows.yaml

```diff
@@ -15,6 +15,7 @@ metrics:
     iis:
       type: iis
       collection_interval: 60s
+      receiver_version: ""
```

Comment:
Nit: the field is already optional and will default to an empty string anyway, so for the sake of keeping the built-in config consistent I'd prefer to leave this out.

Reply:
It's getting updated with the goldens even with omitempty and failing if removed.

Reply:
`omitempty` has to go on the `yaml` tag, not `validate`. With that it works on my end.